package techclallenge5.fiap.com.msGestaoItem.exception;

public class PrecoItemNotFoundException extends RuntimeException {
    public PrecoItemNotFoundException() {
        super("O preco do item n√£o foi encontrado!");
    }
}
